lighthouse-core/audits/image-aspect-ratio.js

```diff
@@ -12,8 +12,27 @@
 'use strict';
 
 const Audit = require('./audit.js');
-
 const URL = require('../lib/url-shim.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the aspect ratios of all images on a site. This descriptive title is shown to users when all images use correct aspect ratios. */
+  title: 'Displays images with correct aspect ratio',
+  /** Title of a Lighthouse audit that provides detail on the aspect ratios of all images on a site. This descriptive title is shown to users when not all images use correct aspect ratios. */
+  failureTitle: 'Displays images with incorrect aspect ratio',
+  /** Description of a Lighthouse audit that tells the user why they should maintain the correct aspect ratios for all images. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Image display dimensions should match natural aspect ratio. ' +
+    '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/aspect-ratio).',
+  /** Warning that the sizing information cannot be collected for an image. */
```

Comment:
```suggestion
  /** Warning that the size information for an image was nonsensical. `url` will be replaced with the url of that image. */
```

Reply:
That was weird, I missed that one, or it wasn't committed? :open_mouth: